firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -207,38 +192,39 @@ def update_notes(sender, instance, **kwargs):
         )
 
 
-# If a site is deleted, delete all docs from index related to site
-@receiver(post_delete, sender=Site)
-def delete_related_docs(sender, instance, **kwargs):
+# Acknowledgement update
+@receiver(post_delete, sender=Acknowledgement)
+@receiver(post_save, sender=Acknowledgement)
+def update_acknowledgement(sender, instance, **kwargs):
     logger = logging.getLogger(ELASTICSEARCH_LOGGER)
-    dictionary_entries_set = instance.dictionaryentry_set.all()
+    dictionary_entry = instance.dictionary_entry
+    acknowledgements_text = get_acknowledgements_text(dictionary_entry)
 
-    for dictionary_entry in dictionary_entries_set:
-        try:
-            existing_entry = get_object_from_index(
-                ELASTICSEARCH_DICTIONARY_ENTRY_INDEX, dictionary_entry.id
-            )
-            if not existing_entry:
-                raise NotFoundError
+    try:
+        existing_entry = get_object_from_index(
+            ELASTICSEARCH_DICTIONARY_ENTRY_INDEX, dictionary_entry.id
+        )
+        if not existing_entry:
+            raise NotFoundError
 
-            dictionary_entry_doc = DictionaryEntryDocument.get(id=existing_entry["_id"])
-            dictionary_entry_doc.delete()
-        except ConnectionError:
-            logger.error(
-                ES_CONNECTION_ERROR
-                % (SearchIndexEntryTypes.DICTIONARY_ENTRY, instance.id)
-            )
-        except NotFoundError:
-            logger.warning(
-                ES_NOT_FOUND_ERROR
-                % (
-                    "sites_delete_signal",
-                    SearchIndexEntryTypes.DICTIONARY_ENTRY,
-                    dictionary_entry.id,
-                )
+        dictionary_entry_doc = DictionaryEntryDocument.get(id=existing_entry["_id"])
+        dictionary_entry_doc.update(acknowledgement=acknowledgements_text)
+    except ConnectionError:
+        logger.error(
+            ES_CONNECTION_ERROR % (SearchIndexEntryTypes.DICTIONARY_ENTRY, instance.id)
```

Comment:
Minor: should this error message be specific that this is an acknowledgement id?